packages/db2/src/connection-manager.ts

```diff
@@ -63,8 +62,8 @@ export class Db2ConnectionManager extends AbstractConnectionManager<Db2Dialect,
 
     try {
       return await new Promise((resolve, reject) => {
-        const connection = new this.lib.Database() as Db2Connection;
-        connection.lib = this.lib;
+        const connection = new this.#lib.Database() as Db2Connection;
+        connection.lib = this.#lib;
```

Comment:
This is not used by anything, do we need to assign?
We can import the required dependency from the library where its required.

```suggestion
```

Reply:
If it's not used for anything, can be removed

If it's currently used, we can import the library from the dialect itself instead (but not directly from the library if we want to re-add the ability to change the connector to say, a fork)

Reply:
It's removed